docs/pages/desktop-access/introduction.mdx

```diff
@@ -1,44 +1,72 @@
+title: Manage Windows resources
+description: Demonstrates how you can manage access to Windows desktops with Teleport.
 videoBanner: n2h0GisWdss
 ---
 
-Teleport manages graphical desktop access to remote hosts. With Teleport, you
-get:
+The topics in this guide describe how to configure Teleport to provide secure, passwordless 
+access to Microsoft Windows desktops and servers. Teleport for Windows desktops provides the
+following key features:
 
-- A password-less login experience backed by strong cryptographic
-  authentication.
-- Role-based access control (RBAC) for groups of hosts and users.
-- Support for copying and pasting to and from remote hosts.
-- Audit log and recording of all desktop connections.
+- Passwordless access to Windows hosts backed by secure cryptographic authentication.
+- Configurable role-based access controls (RBAC) for groups of hosts and users.
+- Configurable clipboard and directory sharing for copying and pasting to and from 
+remote Windows hosts.
+- Session recording for all desktop activity.
+- Audit logs that track user activity.
 
-<Admonition
-  type="note"
-  title="Supported platforms"
->
-  Only Windows hosts accessible over RDP are supported. Specifically:
+Teleport Desktop Access relies on the remote desktop protocol (RDP) to connect to
+remote Windows hosts. Therefore, Teleport Desktop Access only supports the following 
+Windows hosts that are accessible over RDP:
 
-| Teleport Version              | Windows Desktop                                           | Active Directory |
-|-------------------------------|-----------------------------------------------------------|------------------|
-| Open source Teleport          | Windows Server 2012 R2 / Windows 10 or newer              | Required         |
-| Cloud and Enterprise Teleport | Windows Server 2012 R2 / Windows 10 or newer              | Optional         |
+- Windows Server 2012 R2, or later.
+- Windows 10, or later.
 
-</Admonition>
+The following diagram provides a simplified view of the architecture for Teleport
+Desktop Access:
+
+```mermaid
+flowchart TD
+    A(Client) -->|Remote desktop protocol over webSockets| B(Teleport Proxy Service)
+    B <-->|Remote desktop protocol over mutual TLS| C(Teleport Windows Desktop Service)
+    C -->|RDP| D[Windows Server 2012]
+    C -->|RDP| E[Windows 10]
+    C -->|LDAP|F[Domain controller]
+```
+
+It's worth noting that the Teleport Windows Desktop Service implements a minimal set 
+of remote desktop protocol features to minimize security vulnerabilities for Windows
+computers. Because secure access is the top priority, the Windows Desktop Service 
+might not be as performant as other RDP clients. In mos cases, you should use the 
+Teleport Windows Desktop Service to manage access to Windows computers where you
+store or manipulate your most sensitive information, rather than as a direct replacement 
+for tools to provide general purpose access to Windows computers.
```

Comment:
```suggestion
might not be as performant as other RDP clients. 

In most cases, you should use the 
Teleport Windows Desktop Service to manage access to Windows computers where you
store or manipulate your most sensitive information, rather than as a direct replacement 
for tools to provide general purpose access to Windows computers.
```

Breaking up a long paragraph and fixing spelling